package.json

```diff
@@ -3,37 +3,49 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "type": "module",
   "scripts": {
-    "migrate": "npx sequelize-cli db:migrate",
-    "down": "npx sequelize-cli db:migrate:undo",
+    "start": "nodemon --exec babel-node src/index",
```

Comment:
- babel-node should not run in production
- run a normal node process

Reply:
so this feature is already on the develop so while pulling it was also included so i can't change